provider/oauth2.go

```diff
@@ -61,13 +61,12 @@ func initOauth2Handler(p Params, service Oauth2Handler) Oauth2Handler {
 	service.conf = oauth2.Config{
 		ClientID:     service.Cid,
 		ClientSecret: service.Csecret,
-		RedirectURL:  service.redirectURL,
 		Scopes:       service.scopes,
 		Endpoint:     service.endpoint,
 	}
 
 	p.Logf("[DEBUG] created %s oauth2, id=%s, redir=%s, endpoint=%s",
-		service.name, service.Cid, service.endpoint, service.redirectURL)
+		service.name, service.Cid, service.endpoint)
```

Comment:
if redir url composer in another method (see my other comment) will be nice to restore here in debug, as practically setting up provider very often fail due to unexpected redir

Reply:
As we intend to use handler route to make redir url, at initialization point  it is not known what routing path will be used to serve auth handler.  But we can show the pattern of RedirectURL in log. 
E.g. redir=http://127.0.0.1:8080/YOUR_ROUTE/google/callback